blog/docs/releases/announcing-knative-v1-6-release.md

```diff
@@ -0,0 +1,150 @@
+**Authors: [Evan Anderson](https://twitter.com/e_k_anderson) (VMware), Paul Schweigert (IBM)**
+
+**Date: 2022-07-12**
+
+# Announcing Knative 1.6 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in [Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+This release brings a number of smaller improvements to the core Knative Serving and Eventing components, and several improvements to specific plugins, in particular `net-kourier`, `net-gateway-api`, and `eventing-rabbitmq`.
+
+## Table of Contents
+- [Serving](#serving)
+- [Eventing](#eventing)
+- [`kn` CLI](#client)
+- [Knative Operator](#operator)
+- [Networking Extensions](#networking-extensions)
+    - [Kourier plugin](#kourier-plugin)
+    - [Gateway-API plugin](#gateway-api-plugin)
+- [Eventing Extensions](#eventing-extensions)
+    - [Apache Kafka Source](#apache-kafka-source)
+    - [RabbitMQ Broker and Source](#rabbitmq-broker-and-source)
+- [Thank you contributors](#thank-you-contributors)
+
+## Serving [(release notes)](https://github.com/knative/serving/releases/tag/knative-v1.6.0)
```

Comment:
```suggestion
## Serving 

[(release notes)](https://github.com/knative/serving/releases/tag/knative-v1.6.0)
```

Or maybe just remove the release notes link? Putting them on the same line breaks the TOC links above.

(sorry, should've caught that on the first go-round)

Reply:
also needs to be done for Eventing, Client, and Operator

Reply:
Done.